Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.hpp

```diff
@@ -76,6 +76,8 @@ class VertexPerformanceWriter final
     double truthMatchProbMin = 0.5;
     /// Whether information about tracks is available
     bool useTracks = true;
+    /// minimum track weight for track to be considered as part of the fit
+    double minTrkWeight = 0.1;
```

Comment:
I guess if we were to default this to 0.1 we would get the exact same output as before?

Reply:
Not quite: We would have to default this to `0.0` to get the same behavior as before.

Reply:
Ah yeah, of course.